notebooks/06_Serving/06_02_Kfserving.ipynb

```diff
@@ -0,0 +1,493 @@
+    "* We are completely uploaded and fine\n",
+    "< HTTP/1.1 200 OK\n",
+    "< content-length: 204\n",
+    "< content-type: application/json\n",
+    "< date: Fri, 10 May 2019 23:22:04 GMT\n",
+    "< server: envoy\n",
+    "< x-envoy-upstream-service-time: 19162\n",
+    "< \n",
+    "{\n",
+    "    \"predictions\": [\n",
+    "        {\n",
+    "            \"scores\": [0.999115, 9.20988e-05, 0.000136786, 0.000337257, 0.000300533, 1.84814e-05],\n",
+    "            \"prediction\": 0,\n",
+    "            \"key\": \"   1\"\n",
+    "        }\n",
+    "    ]\n",
+    "* Connection #0 to host 34.83.190.188 left intact\n",
+    "}%\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you stop making requests to the application, you should eventually see that your application scales itself back down to zero. Watch the pod until you see that it is `Terminating`. This should take approximately 90 seconds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl get pods --watch -n kubeflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: To exit the watch, use `ctrl + c`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Canary Rollout\n",
+    "\n",
+    "To test a canary rollout, you can use the tensorflow-canary.yaml \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Apply the CRD"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl apply -n kubeflow -f tensorflow-canary.yaml "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To verify if your traffic split percenage is applied correctly, you can use the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl get inferenceservices -n kubeflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output should looks the similar as below:\n",
+    "```\n",
+    "NAME             READY     URL                                  DEFAULT TRAFFIC   CANARY TRAFFIC   AGE\n",
+    "flowers-sample   True      http://flowers-sample.default.example.com   90                10               48s\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "NOTE: KFServing scales pods to 0 in the absence of traffic."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clean Up Resources"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl delete inferenceservices flowers-sample"
```

Comment:
Seems you only delete resource of Yaml? Have you delete SDK example?

Reply:
We have already cleaned up the SDK resources after we finished SDK sample.

![image](https://user-images.githubusercontent.com/23116624/81221246-6d9c1280-8f97-11ea-94e0-0866ad9f623a.png)